packages/backend/src/open_payments/payment/outgoing/lifecycle.ts

```diff
@@ -120,11 +124,7 @@ function getAdjustedAmounts(
   // This is only an approximation of the true amount delivered due to exchange rate variance. Due to connection failures there isn't a reliable way to track that in sync with the amount sent (particularly within ILP payments)
   // eslint-disable-next-line no-case-declarations
   const amountDelivered = BigInt(
-    Math.ceil(
-      Number(alreadySentAmount) *
-        (payment.quote.estimatedExchangeRate ||
-          payment.quote.lowEstimatedExchangeRate.valueOf())
-    )
+    Math.ceil(Number(alreadySentAmount) * payment.quote.estimatedExchangeRate)
```

Comment:
wont have the low `lowEstimatedExchangeRate` to fall back on anymore (it will be optional) so I ensured `estimatedExchangeRate` is always set in the migration. Set it to be `lowEstimatedExchangeRate` where null, just like we're setting it in the ilp `getQoute` method.